api/v1/aerospikecluster_validating_webhook.go

```diff
@@ -222,6 +222,25 @@ func (c *AerospikeCluster) validate(aslog logr.Logger) (admission.Warnings, erro
 		)
 	}
 
+	initVersion, err := GetImageVersion(GetAerospikeInitContainerImage(c))
+	if err != nil {
+		return warnings, err
+	}
+
+	if GetBool(c.Spec.EnableDynamicConfigUpdate) {
+		val, err = lib.CompareVersions(initVersion, minInitVersionForDynamicConf)
+		if err != nil {
+			return warnings, fmt.Errorf("failed to check init image version: %v", err)
+		}
+
+		if val < 0 {
+			return warnings, fmt.Errorf("cannot set enableDynamicConfigUpdate flag, init container version is less"+
+				" than %s. Please visit https://aerospike.com/docs/cloud/kubernetes/operator/Cluster-configuration-settings#spec"+
+				" for more details about enableDynamicConfigUpdate flag",
+				minInitVersionForDynamicConf)
+		}
+	}
```

Comment:
See if we can move this common code to a func and reuse that both in create and update

Reply:
Both ways are different, while create, we check init version in spec and while update we check status and get the minimum version running. And apart from this error message is also different in both cases. I don't think so we will be saving much on code duplication here,